src/app/containers/ChartbeatAnalytics/index.test.jsx

```diff
@@ -91,35 +138,57 @@ describe('Charbeats Analytics Container', () => {
       .fn()
       .mockImplementation(() => 'secction1 section2');
 
+    const toggleState = {
+      live: {
+        chartbeatAnalytics: {
+          enabled: true,
+        },
```

Comment:
Confused here, the spec name says "toggle enabled for chartbeat on test" but we are changing the state for `live` in this fragment. How does this object relate to `mockToggleState`? The variable naming could make this clearer.

Reply:
thanks @jamesdonoh, I think the spec needs to be clearer meant to be `on live`. the `mockToggleState` is a default just in case we don't pass in any `toggleState` would change to `defaultToggleState`